src/go/rdctl/pkg/snapshot/manager.go

```diff
@@ -117,6 +124,16 @@ func (manager Manager) List() ([]Snapshot, error) {
 			return []Snapshot{}, fmt.Errorf("failed to unmarshal contents of %q: %w", metadataPath, err)
 		}
 		snapshot.Created = snapshot.Created.Local()
+
+		completeFileExists := true
+		completeFilePath := filepath.Join(manager.Paths.Snapshots, snapshot.ID, completeFileName)
+		if _, err := os.Stat(completeFilePath); errors.Is(err, os.ErrNotExist) {
```

Comment:
I wonder if checking for `os.ErrNotExists` is the right thing to do here. Practically speaking, it is the only error likely to occur, but just hypothetically, if the error was something cause by incorrect access permissions, why would we treat this as if the file exists?

```suggestion
		if _, err := os.Stat(completeFilePath); err != nil {
```